src/helm/proxy/clients/vertexai_client.py

```diff
@@ -151,25 +189,18 @@ def make_request(self, request: Request) -> RequestResult:
 
         completions: List[Sequence] = []
         model_name: str = request.model_engine
+        model = self.get_model(model_name)
 
         try:
 
             def do_it():
-                model = GenerativeModel(model_name)
```

Comment:
Is `GenerativeModel` a heavy or lightweight object? If the latter, can we just create this in `__init__()` i.e. `self.model = GenerativeModel(model_name)` and have one per instance?

Reply:
I updated it so we cache it after the first request.